pyanaconda/installation.py

```diff
@@ -234,22 +235,17 @@ def _prepare_installation(payload, ksdata):
     installation_queue.task_completed.connect(lambda x: progress_step(x.name))
 
     # This should be the only thread running, wait for the others to finish if not.
```

Comment:
In upstream, could you move this code to a new function and call it called from `run_installation`, please?

Reply:
Sounds good, let's do that in the upstream version. :)

Reply:
changed in upstream PR: #3720